Documents new D4C capabilities (#4501)
elastic/docs-content: solutions/security/cloud/d4c/get-started-with-d4c.md

---
mapped_pages:
  - https://www.elastic.co/guide/en/security/current/d4c-get-started.html
applies_to:
  stack: beta 9.3
  serverless:
    security: beta
---

# Get started with Defend for Containers for Kubernetes [d4c-get-started]

::::{warning}
This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
::::

This page describes how to set up Defend for Containers (D4C) for Kubernetes.

::::{admonition} Requirements
* Kubernetes node operating systems must have Linux kernels 5.10.16 or higher.
* {{stack}} version 9.3 or higher.

::::

## Initial setup [_initial_setup]

First, you’ll need to deploy Elastic’s Defend for Containers integration to the Kubernetes clusters you wish to monitor.

1. Find **Container Workload Security** in the navigation menu or use the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md). Click **Add D4C Integration**.
2. Name the integration. The default name, which you can change, is `cloud_defend-1`.

> 3. Optional: make any desired changes to the integration’s policy by adjusting the **Selectors** and **Responses** sections. (For more information, refer to [Defend for Containers policy](d4c-policies.md)). You can also change these later.

Reviewer: ```suggestion
3. (Optional) Make desired changes to the [integration’s policy](d4c-policies.md) by adjusting the **Selectors** and **Responses** sections. You can change these later, if needed.
```
